packages/mobile/src/screens/notifications-screen/Notifications/USDCPurchaseSellerNotification.tsx

```diff
@@ -47,6 +47,11 @@ export const USDCPurchaseSellerNotification = (
   }, [navigation, notification])
 
   if (!content || !buyerUser) return null
+
+  // Calculate total amount using fixed-decimal
+  const totalAmount = (BigInt(amount) + BigInt(extraAmount)) as UsdcWei
```

Comment:
nit: could do `USDC(BigInt(amount)).value + USDC(BigInt(extraAmount)).value` but this is fine and probably more performant